src/latexdocument.cpp

```diff
@@ -397,6 +400,20 @@ inline bool isDefinitionArgument(const QString &arg)
 	return (pos >= 0 && pos < arg.length() - 1 && arg[pos + 1].isDigit());
 }
 
+/*!
+ * Helper for patchStructure, stores location and name of user-defined commands
+ * \param line line of the definition
+ * \param name name of the command (or environment)
+ * \param start column at which the definition begins
+ */
+void LatexDocument::addUserCommandDefinition(const QDocumentLine line, const QString &name, int start) {
```

Comment:
QDocumentLine as argument seems to be not very useful here as you only use the linehandle.

Reply:
Indeed, updated it to only take the line handle.